tests/queries/0_stateless/00318_pk_tuple_order.sql

```diff
@@ -7,6 +7,7 @@ INSERT INTO pk (x, y, z) VALUES (1, 11, 1235), (1, 11, 4395), (1, 22, 3545), (1,
 
 SET min_insert_block_size_rows = 0, min_insert_block_size_bytes = 0;
 SET max_block_size = 1;
+SET parallel_replicas_index_analysis_only_on_coordinator = 0;
```

Comment:
Please add a comment why the setting is set in every changed test

Reply:
Also, `parallel_replicas_index_analysis_only_on_coordinator` has effect only if `parallel_replicas_local_plan` is enabled, but this setting is randomized in CI. So, both settings should be set to make desired effect reliably

Reply:
Thank you for the reminder. 
However, my intention here is to prevent remote replicas from skipping index analysis. Otherwise, remote replicas may return full ranges, causing `ReadFromMergeTree::AnalysisResult::checkLimits` to fail. 

If I set `parallel_replicas_index_analysis_only_on_coordinator=0`, index analysis will not be skipped regardless of whether `parallel_replicas_local_plan` is true or false.